### PR TITLE
Change [category] and [severity] attribute to classes

### LINKED ISSUE #121: Change [category] and [severity] to classes
This will let us get rid of any `is=""` hacks to be able to use non-standard attributes.  We'll need to update both the new markup, old markup, and CSS all in the same patch.

### PR DESCRIPTION
**Pull request by nchevobbe**
Duplicate CSS rules in order to keep the current frontend working.
Fix #121

### CODE REVIEW

**Comment by linclark** [comment]
Nice, this looks good to me.

@bgrins Once this is commited, the only reason we'd still keep category/severity around AFAIK is in case we wanted to run the old tests against the new code. Do we still want to do that do you think? If not, I'd suggest we have a followup to this one that removes them entirely.

File: devtools/client/webconsole/new-console-output/components/message-types/console-api-call.js [hunk 1]
```diff
@@ -24,22 +24,27 @@ ConsoleApiCall.propTypes = {
...
+  }
+
+  if (severity) {
+    classes.push(severity);
+  }
+
   return dom.div({
-    class: "message cm-s-mozilla",
-    is: "fdt-message",
-    category: message.category,
```
Comment (linclark) [inline devtools/client/webconsole/new-console-output/components/message-types/console-api-call.js:41]: we should make sure to remove this and add the new classes to all message types.

**Comment by bgrins** [comment]
> @bgrins Once this is commited, the only reason we'd still keep category/severity around AFAIK is in case we wanted to run the old tests against the new code. Do we still want to do that do you think? If not, I'd suggest we have a followup to this one that removes them entirely.

I would be happy with a follow up to remove the attributes entirely.  It'll require a bunch of test changes and changing the current console frontend, but it's worthwhile to do

**Comment by linclark** [comment]
Follow-up created.

@nchevobbe can you make sure the new classes are applied to all relevant message-types? Once that is done I think this will be good to go.

**Comment by nchevobbe** [comment]
> can you make sure the new classes are applied to all relevant message-types? Once that is done I think this will be good to go.

@linclark this is done. I'll take care of the follow-up issue when this is merged.

**Comment by linclark** [comment]
Awesome, thanks!

**Pull request merged**